agnosticv-operator/operator/agnosticvcomponent.py

```diff
@@ -111,6 +111,12 @@ def ansible_control_plane(self):
     def ansible_control_plane_secret(self):
         return self.ansible_control_plane.get('secret')
 
+    @property
+    def asset_uuid(self):
+        if 'asset_uuid' in self.__meta__:
+            return self.__meta__['asset_uuid']
+        return str(uuid.uuid5(uuid.NAMESPACE_DNS, self.name.rsplit('.', 1)[0]))
```

Comment:
in case it's not defined, you're building the uuid using the name of the agnosticv-component?

I'm not sure we should do that.
What if the `asset_uuid` is then specified into agnosticv?
That means this function will return something different, right?

The whole point is to have something that never changes (unless we explicitly want to, of course), no?

Reply:
Agreed. We should not generate a default value. It must be explicitly set in AgV.

Reply:
I removed it from the code, it will be empty if we don't have `asset_uuid`.